File: src/main/java/com/clearcont/clearcontapp/repository/ComposicaoDePendenciaRepository.java
package com.clearcont.clearcontapp.repository;

import com.clearcont.clearcontapp.model.ComposicaoDePendencia;
import org.springframework.data.jpa.repository.JpaRepository;
import org.springframework.stereotype.Repository;

@Repository
public interface ComposicaoDePendenciaRepository extends JpaRepository<ComposicaoDePendencia, Integer> {
}

File: src/main/kotlin/br/com/clearcont/clearcontwebapp/repositories/ComposicaoDePendenciaRepository.kt
package br.com.clearcont.clearcontwebapp.repositories

import br.com.clearcont.clearcontwebapp.models.ComposicaoDePendencia
import org.springframework.data.jpa.repository.JpaRepository
import org.springframework.stereotype.Repository

@Repository
interface ComposicaoDePendenciaRepository : JpaRepository<ComposicaoDePendencia, Long>
